### PR TITLE
Minor cleanup (pep8, readability, ReST)

### PR DESCRIPTION
**Pull request by jensens**

### CODE REVIEW

**Comment by jensens** [comment]
http://jenkins.plone.org/job/pull-request-5.0/451/

File: README.rst [hunk 1]
```diff
@@ -4,4 +4,9 @@ plone.intelligenttext
 Overview
 --------
 
-Provides transforms from text/x-web-intelligent to text/html and vice versa.
+Provides transforms from ``text/x-web-intelligent`` to ``text/html`` and vice versa.
+
+Line breaks and indentation is preserved, and web and email addresses are made into clickable links.
```
Comment (adamcheasley) [inline README.rst]: This should read:
"Line breaks and indentation are preserved..."

File: plone/intelligenttext/README.rst [hunk 2]
```diff
@@ -1,24 +1,23 @@
...
 ================
 
-Started by Martin Aspeli
+This package contains a transform that is capable converting plain text into HTML.
 
-This package contains a and a transform (for example for
-portal_transforms in CMF) that is capable converting plain text into
-HTML where line breaks and indentation is preserved, and web and email
-addresses are made into clickable links.
+Line breaks and indentation is preserved, and web and email addresses are made into clickable links.
```
Comment (adamcheasley) [inline plone/intelligenttext/README.rst]: Again here:
"Line breaks and indentation are preserved..."

Comment (jensens) [inline plone/intelligenttext/README.rst]: thats if you copy paste the roiginal text :)

**Comment by jensens** [comment]
travis and jenkins green

**Pull request merged**